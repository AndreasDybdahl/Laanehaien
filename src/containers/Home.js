import React from 'react';
import styles from '../App.css';
import {
  Link
} from 'react-router-dom';

const Home = () => (
  <div className={styles.appIntro}>
    Velkommen til Lånehaien, her kan du låne masse cash for en rolig rente på 46%.
    <br />
    <br />
    <div>
      <Link to="/loan">
        <button className={(styles.btn)}>Jeg vil søke om lån</button>
      </Link>
    </div>
  </div>
)

export default Home;