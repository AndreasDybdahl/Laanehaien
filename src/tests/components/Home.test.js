import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../containers/Home';
import { 
    BrowserRouter as Router,
} from 'react-router-dom';

it('should render the home container', () => {
    const tree = renderer.create(
        <Router>
            <Home />
        </Router>
    ).toJSON();

    expect(tree).toMatchSnapshot();
})