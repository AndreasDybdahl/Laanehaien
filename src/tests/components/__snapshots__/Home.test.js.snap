exports[`test should render the home container 1`] = `
<div
  className={undefined}>
  Velkommen til Lånehaien, her kan du låne masse cash for en rolig rente på 46%.
  <br />
  <br />
  <div>
    <a
      href="/loan"
      onClick={[Function]}>
      <button
        className={undefined}>
        Jeg vil søke om lån
      </button>
    </a>
  </div>
</div>
`;
